src/Mobs/Monster.h

```diff
@@ -217,6 +217,7 @@ class cMonster :
 
 	virtual void HandleFalling(void);
 	int m_LastGroundHeight;
+	int m_JumpCoolDown;
```

Comment:
It's a value which is between 20 and 0, I don't see the merit of an unsigned int here. I could even use an int_8 (perhaps I should?)